diff --git a/simple_field_formatter.module b/simple_field_formatter.module
index eb0bf90..2b1e827 100644
--- a/simple_field_formatter.module
+++ b/simple_field_formatter.module
@@ -19,6 +19,7 @@ function simple_field_formatter_field_formatter_info() {
 				'custom_strtolower' => '',
 				'custom_strtoupper' => '',
 				'custom_ucfirst' => '',
+				'custom_ucwords' => '',
       ),
       // You can usually leave 'multiple values' as follows:
       'multiple values' => FIELD_BEHAVIOR_DEFAULT,
@@ -78,6 +79,12 @@ function simple_field_formatter_field_formatter_settings_form($field, $instance,
     '#description'    => t('Converts the first character of a string to uppercase.'),  // helper text
     '#default_value'  => $settings['custom_ucfirst'],               // Get the value if it's already been set
   );
+	$element['custom_ucwords'] = array(
+    '#type'           => 'checkbox',                        // Use a textbox
+    '#title'          => t('ucwords'),                      // Widget label
+    '#description'    => t('Converts the first character of each word in a string to uppercase.'),  // helper text
+    '#default_value'  => $settings['custom_ucwords'],               // Get the value if it's already been set
+  );
   return $element;
 }
 
@@ -87,7 +94,7 @@ function simple_field_formatter_field_formatter_settings_form($field, $instance,
 function simple_field_formatter_field_formatter_settings_summary($field, $instance, $view_mode) {
   $display = $instance['display'][$view_mode];
   $settings = $display['settings'];
-  $summary = t('<strong>Prefix:</strong> @custom_prefix <br/> <strong>Suffix:</strong> @custom_suffix<br/> <strong>Reverse:</strong> @custom_reverse<br/> <strong>Trim:</strong> @custom_trim<br/> <strong>strtolower:</strong> @custom_strtolower<br/> <strong>strtoupper:</strong> @custom_strtoupper<br/> <strong>ucfirst:</strong> @custom_ucfirst', array(
+  $summary = t('<strong>Prefix:</strong> @custom_prefix <br/> <strong>Suffix:</strong> @custom_suffix<br/> <strong>Reverse:</strong> @custom_reverse<br/> <strong>Trim:</strong> @custom_trim<br/> <strong>strtolower:</strong> @custom_strtolower<br/> <strong>strtoupper:</strong> @custom_strtoupper<br/> <strong>ucfirst:</strong> @custom_ucfirst<br/> <strong>ucwords:</strong> @custom_ucwords', array(
     '@custom_prefix'     => $settings['custom_prefix'],
     '@custom_suffix'  => $settings['custom_suffix'],
 		'@custom_reverse'  => $settings['custom_reverse'],
@@ -95,6 +102,7 @@ function simple_field_formatter_field_formatter_settings_summary($field, $instan
 		'@custom_strtolower'  => $settings['custom_strtolower'],
 		'@custom_strtoupper'  => $settings['custom_strtoupper'],
 		'@custom_ucfirst'  => $settings['custom_ucfirst'],
+		'@custom_ucwords'  => $settings['custom_ucwords'],
   )); // we use t() for translation and placeholders to guard against attacks
   return $summary;
 }
@@ -115,6 +123,7 @@ function simple_field_formatter_field_formatter_view($entity_type, $entity, $fie
 	$custom_strtolower = $settings['custom_strtolower']; 
   $custom_strtoupper = $settings['custom_strtoupper']; 
 	$custom_ucfirst = $settings['custom_ucfirst']; 
+	$custom_ucwords = $settings['custom_ucwords']; 
   
 
   foreach ($items as $delta => $item) {
@@ -135,6 +144,9 @@ function simple_field_formatter_field_formatter_view($entity_type, $entity, $fie
 		if($custom_ucfirst){
 			$original_value = ucfirst($original_value);
 		}
+		if($custom_ucwords){
+			$original_value = ucwords($original_value);
+		}
 		$modified_value = $custom_prefix . $original_value . $custom_suffix;
 		$element[$delta]['#markup'] = $modified_value;
   }
